pom.xml

```
@@ -384,7 +385,7 @@ Import-Package: \\
         <plugin>
           <groupId>org.apache.maven.plugins</groupId>
           <artifactId>maven-surefire-plugin</artifactId>
-          <version>3.0.0-M5</version>
+          <version>3.0.0-M6</version>
```

If it works, I would use:

```suggestion
          <version>3.0.0-M7</version>
```